skills/csharp/emailskill/Deployment/Resources/template.json

```diff
@@ -107,13 +115,12 @@
     "botWebAppName": "[replace(parameters('botWebAppName'), '_', '')]",
     "botServiceName": "[take(parameters('botServiceName'), 42)]",
     "storageAccountName": "[toLower(take(replace(replace(parameters('storageAccountName'), '-', ''), '_', ''), 24))]",
-    "cosmosDbAccountName": "[toLower(take(replace(parameters('cosmosDbName'), '_', ''), 31))]",
-    "botEndpoint": "[concat('https://', toLower(variables('botWebAppName')), '.azurewebsites.net/api/messages')]"
+    "cosmosDbAccountName": "[toLower(take(replace(replace(parameters('cosmosDbName'), '-', ''), '_', ''), 31))]"
   },
   "resources": [
     {
       "apiVersion": "2018-02-01",
-      "name": "df123cff-9cd6-4290-a7a7-113afe2363f7",
+      "name": "99ea37e6-a3e6-4102-a249-71c880607386",
```

Comment:
what's the meaning of this name? They are all different before

Reply:
looks like some skills have the same name with the sample while others not. Anyway, Kept it as original